Requirement already satisfied: statsmodels in ./.conda/envs/Tesis3/lib/python3.9/site-packages (0.14.2)
Requirement already satisfied: numpy>=1.22.3 in ./.local/lib/python3.9/site-packages (from statsmodels) (1.23.0)
Requirement already satisfied: scipy!=1.9.2,>=1.8 in ./.local/lib/python3.9/site-packages (from statsmodels) (1.8.0)
Requirement already satisfied: pandas!=2.1.0,>=1.4 in ./.conda/envs/Tesis3/lib/python3.9/site-packages (from statsmodels) (2.2.2)
Requirement already satisfied: patsy>=0.5.6 in ./.conda/envs/Tesis3/lib/python3.9/site-packages (from statsmodels) (0.5.6)
Requirement already satisfied: packaging>=21.3 in ./.conda/envs/Tesis3/lib/python3.9/site-packages (from statsmodels) (24.0)
Requirement already satisfied: python-dateutil>=2.8.2 in ./.conda/envs/Tesis3/lib/python3.9/site-packages (from pandas!=2.1.0,>=1.4->statsmodels) (2.9.0.post0)
Requirement already satisfied: pytz>=2020.1 in ./.conda/envs/Tesis3/lib/python3.9/site-packages (from pandas!=2.1.0,>=1.4->statsmodels) (2024.1)
Requirement already satisfied: tzdata>=2022.7 in ./.conda/envs/Tesis3/lib/python3.9/site-packages (from pandas!=2.1.0,>=1.4->statsmodels) (2024.1)
Requirement already satisfied: six in ./.conda/envs/Tesis3/lib/python3.9/site-packages (from patsy>=0.5.6->statsmodels) (1.16.0)
Requirement already satisfied: pyyaml in ./.conda/envs/Tesis3/lib/python3.9/site-packages (6.0.1)
Requirement already satisfied: scikit-learn in ./.conda/envs/Tesis3/lib/python3.9/site-packages (1.4.2)
Requirement already satisfied: numpy>=1.19.5 in ./.local/lib/python3.9/site-packages (from scikit-learn) (1.23.0)
Requirement already satisfied: scipy>=1.6.0 in ./.local/lib/python3.9/site-packages (from scikit-learn) (1.8.0)
Requirement already satisfied: joblib>=1.2.0 in ./.conda/envs/Tesis3/lib/python3.9/site-packages (from scikit-learn) (1.4.2)
Requirement already satisfied: threadpoolctl>=2.0.0 in ./.conda/envs/Tesis3/lib/python3.9/site-packages (from scikit-learn) (3.5.0)
Soy un JOB de prueba
Corri en la maquina: nodei-10
Corri el: nodei-10
Ejecutando Script de python 

slurmstepd: error: *** JOB 218449 ON nodei-10 CANCELLED AT 2024-05-17T20:18:03 ***
